controllers/dbaas.redhat.com/suite_test.go

```diff
@@ -18,13 +18,14 @@ package dbaasredhatcom
 
 import (
 	"context"
+	"os"
```

Comment:
Please make/keep includes consistent with (core <line break> 3rd party <line break> org/local/project ) grouping used in the other go files. (one step in the right direction would use `goimports` on save over `gofmt`, as the former is a superset)